projects/plugins/crm/modules/woo-sync/jpcrm-woo-sync-init.php

```diff
@@ -243,6 +243,17 @@ function jpcrm_woosync_is_hpos_enabled() {
 	return false;
 }
 
+// If we ever have more WooSync constants we should create a separate php file.
+if ( ! defined( 'WOOSYNC_DO_NOT_CREATE' ) ) {
+	define(
+		'WOOSYNC_DO_NOT_CREATE',
```

Comment:
For this global definition I'd add the `JPCRM_` prefix => `JPCRM_aWOOSYNC_DO_NOT_CREATE`